src/Components/ArtworkFilter/index.tsx

```diff
@@ -138,21 +148,6 @@ export const BaseArtworkFilter: React.FC<
     "diamond_revised-artwork-filters"
   )
 
-  /**
-   * Check to see if the mobile action sheet is present and prevent scrolling
-   */
-  useEffect(() => {
-    const setScrollable = doScroll => {
-      document.body.style.overflowY = doScroll ? "visible" : "hidden"
-    }
-    if (showMobileActionSheet) {
-      setScrollable(false)
-    }
-    return () => {
-      setScrollable(true)
-    }
-  }, [showMobileActionSheet])
```

Comment:
This didn't do anything since we switched to the new modal (a very long time ago), which handles it's own scroll locking.